File: webApp/src/app/common/bus.ts
/**
 * 汽车实体
 */
export class Bus {
  /** id */
  id: number;

  /** 车牌号 */
  busNumber: string;

  /** 最大座位数 */
  seatNumber: number;

  constructor(data?: { id?: number, busNumber?: string, seatNumber?: number }) {
    if (data) {
      this.id = data.id;
      this.busNumber = data.busNumber;
      this.seatNumber = data.seatNumber;
    }
  }
}

File: bus-ticket/src/main/java/xiaoqiangZzz/busticket/Entity/Bus.java
package xiaoqiangZzz.busticket.Entity;

import com.mengyunzhi.core.entity.YunzhiEntity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Bus implements YunzhiEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    /**
     * 车牌号
     */
    private String busNumber;

    /**
     * 车牌号
     */
    private int seatNumber;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getBusNumber() {
        return busNumber;
    }

    public void setBusNumber(String busNumber) {
        this.busNumber = busNumber;
    }

    public int getSeatNumber() {
        return seatNumber;
    }

    public void setSeatNumber(int seatNumber) {
        this.seatNumber = seatNumber;
    }
}
